ra_aid/provider_strategy.py

```
@@ -213,6 +213,32 @@ def validate(self, args: Optional[Any] = None) -> ValidationResult:
 
         return ValidationResult(valid=len(missing) == 0, missing_vars=missing)
 
+class GeminiStrategy(ProviderStrategy):
+    """Gemini provider validation strategy."""
+
+    def validate(self, args: Optional[Any] = None) -> ValidationResult:
```

Ruff (0.8.2)

Dynamically typed expressions (typing.Any) are disallowed in `args`

(ANN401)